components/hab/src/cli/hab/bldr.rs

```diff
@@ -77,6 +83,9 @@ pub struct JobStatus {
 }
 
 /// Cancel a build job group and any in-progress builds
+///
+/// NOTICE: Public Builder Build Functions are no longer supported. Please reach
+/// out to your account team if you were using these features.
```

Comment:
Should it be if you **are** using these features? instead of you **were** ?

Reply:
nitpick: Also, this text is not exactly identical to one mentioned in the Jira issue. It should be using ***this feature*** and not ***these features***.

Reply:
mesg now matches Chaithra's.